hackathon/models.py

```diff
@@ -63,3 +63,34 @@ class HackathonEvent(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
 
+<<<<<<< HEAD
```

Comment:
Resolve conflicts ... this needs to understood first and then do it
Link https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line